src/Sylius/Bundle/CoreBundle/Resources/views/Email/userConfirmation.html.twig

```diff
@@ -1,6 +1,10 @@
 {% block subject %}
-    {% autoescape false %}
-    {{ 'email.customer_welcome.subject'|trans }}
-    {% endautoescape %}
+    {{ 'sylius.ui.welcome_to_our_store'|trans }}
+{% endblock %}
+
+{% block body %}
+    {{ 'sylius.ui.welcome_to_our_store'|trans }}
```

Comment:
All e-mail bodies should be just ``sylius.email.<type>.content``. So that you can only customize the translation key to change the message, without overriding the template.